src/components/SimpleRadioGroupField.tsx

```diff
@@ -0,0 +1,59 @@
+import { ChangeEvent } from "@dashboard/hooks/useForm";
+import { RadioGroup, Text } from "@saleor/macaw-ui-next";
+import React from "react";
+
+type RadioGroupFieldChoice = {
+  label: string | React.ReactNode;
+  value: string;
+  disabled?: boolean;
+};
+
+interface SimpleRadioGroupFieldProps {
+  name: string;
+  value: string;
+  error: boolean;
+  onChange: (event: ChangeEvent) => void;
+  choices: RadioGroupFieldChoice[];
+  errorMessage?: string;
+}
+
+export const SimpleRadioGroupField: React.FC<SimpleRadioGroupFieldProps> = ({
```

Comment:
question: What does mean that this component is simple? :D

Reply:
`RadioGroupField` was already taken 😄 (it uses MUI) it had a bit more functionalities compared to this one

Reply:
Here you have example how to do this https://github.com/saleor/saleor-dashboard/blob/main/src/components/Select/Select.tsx